src/serviceWorker.ts

```diff
@@ -28,10 +26,7 @@ type Config = {
 export function register(config?: Config) {
   if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
     // The URL constructor is available in all browsers that support SW.
-    const publicUrl = new URL(
-      process.env.PUBLIC_URL,
-      window.location.href,
-    );
+    const publicUrl = new URL(process.env.PUBLIC_URL!, window.location.href);
```

Comment:
prettier가 함께 돌면서 많이 바뀌었는데, 버그 수정의 핵심은 이 라인입니다.

Reply:
env에 PUBLIC_URL을 추가할 필요는 없는건가요?

Reply:
env 에 `PUBLIC_URL` 이 없을 경우가 있기 때문에 `!` 를 붙이는건 안전하지 않은거 같아요! 아래처럼 하는건 어떨까 싶어요!

```suggestion
    const publicUrl = new URL(process.env.PUBLIC_URL ?? '', window.location.href);
```

차피 뒤에 `baseURL` 이 들어가니까 `PUBLIC_URL` 이 `null` 이면 빈 문자열을 넣어도 될거같아용